core/src/main/java/nl/nn/adapterframework/extensions/fxf/FxfWrapperPipe.java

```diff
@@ -240,21 +244,27 @@ private int retrieveStartTransferVersion() {
 		return 0;
 	}
 
+	public void setFlowId(String flowId) {
+		this.flowId = flowId;
+	}
 	public String getFlowId() {
 		return flowId;
 	}
 
-	public void setFlowId(String flowId) {
-		this.flowId = flowId;
+	public void setFilePath(String filePath) {
```

Comment:
Moeten we dit nu filePath noemen? volgens mij heet het bij filesystems `rootFolder`, bij de filepipe `directory` en nu weer iets anders?